src/classifier/Features.h

```diff
@@ -50,4 +55,76 @@ class Features {
   Pipe *pipe_; // The pipe that owns this feature handler.
 };
 
+
+//struct  FeatureLabelPair : Structure to define a a feature-label pair
+struct FeatureLabelPair {
+	uint64_t feature;
```

Comment:
Never use tabs, use spaces (with the same indentation level). There should be no single tab in any *.cpp and *.h files.